src/SFA.DAS.EmployerFinance.Web/Extensions/UrlHelperExtensions.cs

```diff
@@ -14,6 +14,19 @@ public static string EmployerAccountsAction(this UrlHelper helper, string path,
             return includedAccountId ? AccountAction(helper, baseUrl, path) : Action(baseUrl, path);
         }
 
+        public static string AccountsAction(this UrlHelper helper, string controller, string action, bool includedAccountId = true)
```

Comment:
Made the same comment on Kevins PR, should this be EmployerAccountsAction() and simply be an overload?